.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -80,7 +81,26 @@ phases:
           -e OFFER_NAME="Ubuntu" \
           ${CONTAINER_IMAGE} make  -f packer.mk generate-sas
         displayName: Getting Shared Access Signature URI
-        condition: eq(variables.DRY_RUN, 'False')
+        condition: and(eq(variables.DRY_RUN, 'False'), eq(variables.HYPERV_GENERATION, 'V1'))
+      - script: |
+          subscription_id="$(cat vhdbuilder/packer/settings.json | grep "subscription_id" | awk -F':' '{print $2}' | awk -F'"' '{print $2}')" && \
```

Comment:
Why are we generating these here and passing them to the pacler, can't we just pass them in as input variables

Reply:
we can but we're currently not passing sub_id as input parameter. Instead it's retrieved in the init-variables.sh

SUBSCRIPTION_ID="${SUBSCRIPTION_ID:-$(az account show -o json --query="id" | tr -d '"')}"

If we want we can refactor both, but I don't feel it does not worthy the effort.

Reply:
Give it a second thought - you're right, the value is already there. Will fix it in the next PR.